snuba/datasets/__init__.py

```diff
@@ -22,6 +22,19 @@ def get_schema(self):
     def get_processor(self):
         return self.__processor
 
+    def get_writer(self, options=None):
+        from snuba.clickhouse import ClickhousePool
+        from snuba.writer import NativeDriverBatchWriter
+
+        pool_opts = {}
+        if options is not None:
+            pool_opts['client_settings'] = pool_opts
+
+        return NativeDriverBatchWriter(
+            self._schema,
+            ClickhousePool(**pool_opts),
```

Comment:
Also, I'm pretty sure we don't really require this pool because we don't do this in a threaded environment.

Reply:
Am I missing something or you are not loading host and port from config anymore ?

Reply:
ah no nevermind. ClickhousePool is defaulting to settings

Reply:
> ah no nevermind. ClickhousePool is defaulting to settings

Yeah, this drives me a little crazy. 🙂